tests/elements-regression/playwright.config.ts

```diff
@@ -1,32 +1,42 @@
-import { config as _config } from 'dotenv';
 import { resolve } from 'path';
 import { defineConfig } from '@playwright/test';
+import { config as _config } from 'dotenv';
+
+process.env.DEV_SERVER = 'http://localhost:8888';
+process.env.TEST_SERVER = 'http://localhost:8888'; // Test server is the same as dev server in elements-regression
+process.env.DEBUG_PORT = ( 1 === Number( process.env.TEST_PARALLEL_INDEX ) ) ? '9223' : '9222';
```

Comment:
I saw this grouping parentheses in various places, they may be redundant.
```suggestion
process.env.DEBUG_PORT = 1 === Number( process.env.TEST_PARALLEL_INDEX ) ? '9223' : '9222';
```

Reply:
Fixed: https://github.com/elementor/elementor/pull/27669/commits/c46c0be3ca9d25f0b0eb82f5e26563fad49206ea